server/challenges/index.js

```diff
@@ -1,23 +1,25 @@
+const config = require('../../config')
+const path = require('path')
+
+let challenges
+
+const resetChallenges = () => {
+  const module = path.join('../../', config.rDeployDirectory, 'config.json')
+
+  delete require.cache[require.resolve(module)]
+  challenges = require(module)
```

Comment:
`require` will cache the module. If we want to add no-downtime challenge reloading, we would have to use `fs.readFile`.